packages/sui-domain/src/EntryPointFactory.js

```diff
@@ -40,10 +40,12 @@ export default ({useCases, config}) =>
       return loader === undefined
         ? new NotImplementedUseCase(key)
         : {
-            execute: params => {
+            execute: (...params) => {
```

Comment:
I don't think that this could be a good approach. We must continue using named params over positional params.
And do whatever you want in your use case

Reply:
why?